src/Microsoft.ML.Data/Transforms/KeyToValue.cs

```diff
@@ -502,6 +499,24 @@ public override JToken SavePfa(BoundPfaContext ctx, JToken srcToken)
         }
     }
 
+    /// <summary>
+    /// Utilizes KeyValues <see cref="AnnotationInfo"/> of the input column, to map keys to the corresponding values.
+    /// Maps zero values of the <see cref="KeyDataViewType"/> are mapped to the <see langword="default"/> value of the output type.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | [KeyDataViewType](xref:Microsoft.ML.Data.KeyDataViewType) |
```

Comment:
>KeyDataViewType [](start = 36, length = 15)

so here we can use 'key type' as the anchor text. we're using raw types in the params requirement instead of IDV types. in this table we have a change to link to the actual class, which explains what we mean by 'key type'.

we're doing the same for text.
[text](xref:System.ReadOnlyMemory{System.Char})

Reply:
how about numeric.. if we keep it to that, we'd have to spell out: float, double, int, long, uint, short, ushort.. doesn't seem feasible. How about i leave it to no references for text and numeric?

---
In reply to: [276847010](https://github.com/dotnet/machinelearning/pull/3394#discussion_r276847010) [](ancestors = 276847010)